.gitignore

```diff
@@ -39,3 +39,6 @@ nosetests.xml
 
 .cache
 htmlcov
+
+node_modules
+package-lock.json
```

Comment:
why is package.lock gitignored?

Reply:
Because we don't really need it here right now. There are no actual JS dependencies, just a handful of dev deps that aren't even run in CI.